notes/lattice_cryptography.tex

```diff
@@ -128,28 +127,29 @@ \subsection{Introduction to Lattices}
 \end{problem}
 
 \noindent It is easy to see how the decision version can be solved using the search version assuming $\chi$ is distinguishable from a uniform distribution over $\Z_q$. Intuitively, given a solver $\advA$ for $\searchlwe$, we can construct $\advA'$ that solves $\decisionlwe$ as follows: $\advA'$ feeds its input into $\advA$. Compute $\varvect{e} = \varvect{A}\varvect{s} - \varvect{D}_b$ using the $\varvect{s}$ returned by $\advA$. When $b=0$, $\varvect{e}$ will be a uniformly random vector from $\Z^m_q$ and when $b=1$, $\varvect{e}$ will be from $\chi^m$. Perhaps surprisingly, under some restrictions, the search variant can also be solved using the decision variant. Oded Regev showed in \cite{Regev2005} that the search and decision variants of the LWE problem are equivalent when $q$ is a prime that is polynomial in $n$. This result was extended in \cite{Peikert2009} for any $q$ that is a product of distinct primes that are polynomial in $n$.
 
 \begin{remark}
-If $\varvect{e}$ is uniform over $\Z^m_q$, then finding $\varvect{s}$ is impossible. At the other extreme, if the error is 0, then the problem reduces to $m$ equations in $n$ unknowns which can be solved using Gaussian elimination to obtain a unique solution when $m \geq n$
+If $\varvect{e}$ is uniform over $\Z^m_q$, then finding $\varvect{s}$ is impossible following the same intuition as the one-time pad. At the other extreme, if $\vece = \textbf{0}$, then the problem reduces to $m$ equations in $n$ unknowns which can be solved using Gaussian elimination to obtain a unique solution when $m \geq n$. In between, when the error is drawn from an uneven Discrete Gaussian distribution, the solution becomes computationally intractable to find. If one attempts to use row reduction anyway (despite the presence of a nonzero error), the linear combination operations performed repeatedly on every row causes the errors to accumulate.
 \end{remark}
 
 \subsection{LWE as a lattice problem and Regev's construction}
-The connection between LWE and lattices stems from the fact that 
-the LWE problem can be viewed as a lattice problem. For certain range of parameters, worst-case hardness of $\gapsvpy$ implies LWE. Regev \cite{Regev2005} built a public key encryption scheme for single bit messages that tightly reduces to the $\decisionlwe$ problem. The details of Regev's construction are given below.
+As very briefly touched upon above, for certain ranges of parameters, worst-case hardness of $\gapsvpy$ implies LWE. This reduction applies for both quantum algorithms \cite{Regev2005} and classical algorithms (the latter as long as $q \ge 2^n$) [\textbf{Peikert '09}]. Regev \cite{Regev2005} built a public key encryption scheme for single bit messages that tightly reduces to the $\decisionlwe$ problem. The details of Regev's construction are given below.
```

Comment:
from lecture: covering that the reduction/implication applies to both classical and quantum algorithms, which were shown in separate papers